portals/publisher/source/src/app/components/Apis/Details/Policies/GatewaySelector.tsx

```diff
@@ -76,32 +77,32 @@ const GatewaySelector: FC<GatewaySelectorProps> = () => {
                             <FormControl component='fieldset'>
                                 <RadioGroup
                                     aria-label='gateway'
-                                    name='gateway-selector-radio-buttons-group'
                                     row
+                                    aria-labelledby="gateway-selector-radio-buttons-group-label"
+                                    name="gateway-selector-radio-buttons-group"
+                                    defaultValue={SupportedGatewayTypes.REGULAR}
                                 >
                                     <FormControlLabel
-                                        // value={SupportedGatewayTypes.REGULAR}
-                                        defaultChecked
-                                        control={(
-                                            <Radio
-                                                color='primary'
+                                        value={SupportedGatewayTypes.REGULAR}
+                                        control={
+                                            <Radio color='primary'
+                                                defaultChecked
                                                 disabled={isRestricted(['apim:api_create'], apiFromContext)}
+                                                onChange={() => getGatewayType(false)}
```

Comment:
It's better (performance-wise) if we can pass onChange handler function 
I:e
```js
onChange={setGatewayType}
```

and use the `value` coming in the fired event to determine the gateway type, As per this case 
set it to `false` if value is `SupportedGatewayTypes.REGULAR` , `true` if value is `SupportedGatewayTypes.CC`

Reply:
Perf problem is

```js
onChange={() => getGatewayType(true)}
```
this line will create a new function per React renders